purge-docker-cluster.yml

```diff
@@ -439,28 +463,32 @@
       ansible_pkg_mgr == "dnf" and
       not is_atomic
 
-  - name: remove six
-    pip:
-      name: six
-      version: 1.9.0
-      state: absent
-    when: not is_atomic
+  - name: remove package dependencies on redhat
+    command: yum -y autoremove
```

Comment:
That's because the yum module does not support the autoremove command. I am going to file a feature request on ansible for that.